app/views/conferences/_conference_details.html.haml

```diff
@@ -34,6 +34,8 @@
               = link_to "My Proposals", conference_program_proposals_path(conference.short_title), class: 'btn btn-default'
             - elsif can? :new, conference.program.events.new
               = link_to "Submit Proposal", new_conference_program_proposal_path(conference.short_title), class: 'btn btn-default'
+            - if current_user && current_user.ticket_purchases.by_conference(conference).unpaid.any?
```

Comment:
Then just add the pysical tickets with the ticket name, and the unpaid ones are reachable from there, as we want a link from the pysical ticket index to the unpaid tickets. So we don't overload it. But this is not related to this PR, so in a different PR please :wink: